proto/crm/rpc/correspondence_svc.proto

```diff
@@ -23,11 +22,27 @@ message CorrespondenceServiceDocumentResponse {
 	string document = 1;
 }
 
+enum CorrespondenceServiceEmailSendStatus {
+	EMAIL_SEND_STATUS_SUCCESS = 0;
+	EMAIL_SEND_STATUS_SERVER_FAILURE = 1;
+	EMAIL_SEND_STATUS_UNKNOWN_FAILURE = 2;
+	EMAIL_SEND_STATUS_INVALID_RECIPIENT = 3;
+}
+
+message CorrespondenceServiceEmailSendResult {
+	CorrespondenceServiceEmailSendStatus send_status = 1;
+	string recipient_email = 2;
```

Comment:
is this an email or an email address? if it's an address i'd rather call it that.